src/backend/input.rs

```diff
@@ -216,6 +231,9 @@ pub trait InputBackend: Sized {
     /// Get current `InputConfig`
     fn input_config(&mut self) -> &mut Self::InputConfig;
 
+    /// Called to inform the Input backend about a new focused Output for a `Seat`
+    fn set_output_metadata(&mut self, seat: &Seat, output: &Output);
```

Comment:
For both touchscreen and tablet devices the screen size is still important in that case.

I would argue, that the usual expected behavior is to have one focused output per seat. When a tablet is used, wouldn't you just move the focus to the fixed output in that case? In that case the only API error is, that you cannot change the output metadata anymore, once you have received the event.

Maybe we should change the `on_pointer` callback then to receive an event like the `TouchEvent` we already have? Then those could implement something like `x_transformed` themselves instead of handling this internally in the backend.